src/common/Connection.ts

```diff
@@ -48,7 +48,7 @@ export const ConnectionEmittedEvents = {
 } as const;
 
 /**
- * @internal
+ * @public
```

Comment:
we should also add TSDoc comments to the methods on this class that are documented on pptr.dev (https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-cdpsessionconnection). Could you look at that in a future PR, please?